config/application.rb

```diff
@@ -14,7 +14,6 @@
 # require "action_cable/engine"
 # require "sprockets/railtie"
 require "rails/test_unit/railtie"
-require "view_component/engine"
```

Comment:
This was a deprecation warning already present, might as well clean up the warning